SparkPost/2web2ui
# SE-477: Blacklist Alerts Create & Edit

See [SE-477](https://jira.int.messagesystems.com/browse/SE-477)

### What Changed
 - New "Blacklists" alert metric option

<img width="1393" alt="Screen Shot 2020-01-15 at 11 10 29 AM" src="https://user-images.githubusercontent.com/1335605/72450046-ab2a2500-3787-11ea-93ce-b987b1d18c82.png">

### How To Test

**Please note that the API is ready, so we can't test everything** 

- Set the account option
- Navigate to /alerts
- Click "Create an Alert"
- Select "Blacklists" from "Alert Metric" dropdown
- Fill-out rest of the form
- Click "Creeate Alert"
